app/Http/Resources/ImageResource.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class ImageResource extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return array
+     */
+    public function toArray($request)
```

Comment:
probably :array() can be added.

Isn't it should be public function toArray( Request $request)?
I can see, in the comments it is the Request object.

Reply:
I think, it needs a refactor. We might need a common Resource class other than Success And Error Resource and then return all the resources from there. We can also use interface and  then extends its implementation.